conf/modules.config

```diff
@@ -36,6 +36,17 @@ process {
         ]
     }
 
+    withName: 'BIOAWK' {
+        publishDir = [
+            path: { "${params.outdir}/bioawk" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.prefix =  { "${meta.id}_rrna.gtf" }
+        // Should we also have \$9 ~ /transcript_type "rRNA"/ as part of the filtering? See 's3://nf-core-awsmegatests/rnafusion/references/GRCh38/gencode_v46/gencode/Homo_sapiens_GRCh38_46.gtf' structure.
+        ext.args   = """-c gff '\$9 ~ /transcript_biotype "rRNA"/ || \$9 ~ /transcript_biotype "Mt_rRNA"/ || \$9 ~ /transcript_biotype "rRNA_pseudogene"/'"""
+    }
```

Comment:
Actually I don't see a transcript_biotype in the gtf s3://nf-core-awsmegatests/rnafusion/references/GRCh38/gencode_v46/gencode/Homo_sapiens_GRCh38_46.gtf
Am i missing something?